odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java

```diff
@@ -0,0 +1,153 @@
+package cz.cuni.mff.xrg.odalic.tasks.annotations;
+
+import java.util.Map;
+import java.util.Set;
+
+import javax.annotation.concurrent.Immutable;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+import cz.cuni.mff.xrg.odalic.api.rest.adapters.StatisticalAnnotationAdapter;
+import cz.cuni.mff.xrg.odalic.api.rest.values.ComponentTypeValue;
+
+/**
+ * Annotates column in a table for statistical data processing.
+ * 
+ * @author Josef Janoušek
+ *
+ */
+@Immutable
+@XmlJavaTypeAdapter(StatisticalAnnotationAdapter.class)
+public final class StatisticalAnnotation {
+
+  private final Map<KnowledgeBase, ComponentTypeValue> component;
```

Comment:
@josef-janousek Oh, nevermind. I got it. It is indexed by a column index, right?